tests/network.spec.coffee

```diff
@@ -1,6 +1,6 @@
 m = require('mochainon')
 _ = require('lodash')
-network = require('../lib/network')
+network = require('../build/network')
```

Comment:
What's the reasoning for testing against uncommitted code?

Reply:
It's the code that will be published and run by downstream users so it makes sense to test that imo rather than the source versions that will never be directly run